src/gui/accountstate.cpp

```diff
@@ -47,47 +47,30 @@ Q_LOGGING_CATEGORY(lcAccountState, "gui.account.state", QtInfoMsg)
 
 // Returns the dialog when one is shown, so callers can attach to signals. If no dialog is shown
 // (because there is one already, or the new URL matches the current URL), a nullptr is returned.
-QDialog *AccountState::updateUrlDialog(const QUrl &newUrl)
+UpdateUrlDialog *AccountState::updateUrlDialog(const QUrl &newUrl)
 {
     // guard to prevent multiple dialogs
     if (_updateUrlDialog) {
         return nullptr;
     }
-    auto accept = [=] {
+
+    _updateUrlDialog = new UpdateUrlDialog(
```

Comment:
I guess it would make sense to move most of the text to the dialog itself, only then its possible to use it elsewhere

Reply:
Is the new wizard going to use the exactly same wording?